### DISCUSSION TITLE
Profile Views Count Bug

### DISCUSSION THREAD

**Discussion by hifarhanali** [community/community#18046] [Other Feature Feedback, Questions, & Ideas]
Today while creating a `README.md` file for my Github profile, I noticed that there is something wrong with **Github Profile Views Count**. I am not sure whether it's a bug or a feature, each time someone refreshes my profile page, my profile views count increases by 1. Even if the same person from the same machine at the same time is refreshing the page, the **profile view count** increases. 

It may not seem like a big issue at first glance, but it can be in the future. Waiting for the updates :)

**Comment by shemsd621** [comment]
The count increases by 1 each time anyone refreshes your profile page and even if it's viewed by the same person multiple times. 

Although this is the case, you can get insights on how many unique visitors viewed your README.md file through Insights>Traffic.

Check this link -> https://www.quora.com/Is-there-a-better-way-to-know-who-visited-my-Github

But yes, a better way to access it would be great!

**Comment by melvincwng** [comment]
@hifarhanali, its not a bug.

The profile views increases by 1 everytime a user visits your page, be it the same or a different user.

Hope this clarifies.

**Comment by ZekeScript** [comment]
I refresh one of mine repos and stay in 1... 🤔
